numba/stencils/stencilparfor.py

```diff
@@ -260,13 +266,142 @@ def _mk_stencil_parfor(self, label, in_args, out_arr, stencil_ir,
             dtype_attr_assign = ir.Assign(dtype_np_attr_call, dtype_attr_var, loc)
             init_block.body.append(dtype_attr_assign)
 
-            stmts = ir_utils.gen_np_call("full",
-                                       np.full,
+            stmts = ir_utils.gen_np_call("empty",
+                                       np.empty,
                                        out_arr,
-                                       [shape_var, zero_var, dtype_attr_var],
+                                       [shape_var, dtype_attr_var],
                                        self.typingctx,
                                        self.typemap,
                                        self.calltypes)
+            # ------------------
```

Comment:
I guess all this logic can be reused in the case where a use supplies an output too (not for this PR though)?

Reply:
Yes.